summingbird-online/src/main/scala/com/twitter/summingbird/online/executor/AsyncBase.scala

```diff
@@ -16,14 +16,15 @@ limitations under the License.
 
 package com.twitter.summingbird.online.executor
 
+import java.util.concurrent.TimeoutException
 import java.util.concurrent.atomic.AtomicInteger
 
 import com.twitter.summingbird.online.Queue
-import com.twitter.summingbird.online.option.{ MaxWaitingFutures, MaxFutureWaitTime, MaxEmitPerExecute }
-import com.twitter.util.{ Await, Future }
-import scala.util.{ Try, Success, Failure }
-import java.util.concurrent.TimeoutException
-import org.slf4j.{ LoggerFactory, Logger }
+import com.twitter.summingbird.online.option.{MaxEmitPerExecute, MaxFutureWaitTime, MaxWaitingFutures}
```

Comment:
we use scalariform to fix the formatting. The goal is not to fight it.

:)

Maybe we should move to scalafmt:
https://github.com/olafurpg/scalafmt